diff --git a/features/bootstrap/FeatureContext.php b/features/bootstrap/FeatureContext.php
index 023a5c4..b4b1a5a 100644
--- a/features/bootstrap/FeatureContext.php
+++ b/features/bootstrap/FeatureContext.php
@@ -3,11 +3,12 @@
 use Behat\Behat\Context\SnippetAcceptingContext;
 use Behat\Gherkin\Node\PyStringNode;
 use Behat\Gherkin\Node\TableNode;
+use Behat\MinkExtension\Context\MinkContext;
 
 /**
  * Behat context class.
  */
-class FeatureContext implements SnippetAcceptingContext
+class FeatureContext extends MinkContext implements SnippetAcceptingContext
 {
     private $output;
 
