diff --git a/features/product_admin.feature b/features/product_admin.feature
index b385a48..c71e87d 100644
--- a/features/product_admin.feature
+++ b/features/product_admin.feature
@@ -2,3 +2,9 @@ Feature: Product admin
   In order to manage the content on my site
   As an admin
   I need to be able to add, edit and delete products
+
+  Scenario: Seeing a list of existing products
+    Given I am on "/"
+    When I follow "Admin"
+    And I follow "Products"
+    Then I should see "Showing All Products"
