diff --git a/features/product_admin.feature b/features/product_admin.feature
index 07654e3..1875697 100644
--- a/features/product_admin.feature
+++ b/features/product_admin.feature
@@ -8,3 +8,12 @@ Feature: Product admin
     When I click "Admin"
     And I click "Products"
     Then I should see "Showing All Products"
+
+  Scenario: Add a new product
+    Given I am on "/products"
+    When I click "new-product-link"
+    And I fill in "Name" with "New prod"
+    And I fill in "Price" with "9.99"
+    And I fill in "Body" with "cool new product!"
+    And I press "Save"
+    Then I should see "Product created FTW"
